packages/apollo-angular/testing/tests/operation.spec.ts

```diff
@@ -52,4 +59,41 @@ describe('TestOperation', () => {
       heroes: [],
     });
   });
+
+  test('should close the operation except for subscription', done => {
```

Comment:
This tests what happens with a subscription, but it does not test what happen with non-subscriptions (query and mutation). Please add tests for those cases too.

Reply:
Aye! I refactored the tests to check each case individually. Sadly I do have low confidence if this is what you expected, please let me know in detail If I have to adapt the test.